### DISCUSSION TITLE
target_lang 具体的字典对应表在哪可以查询？

### DISCUSSION THREAD

**Discussion by piwawa** [zh-plus/openlrc#35] [Q&A] [answered]
lrcer.run('./data/test.mp3', target_lang='zh-cn')
如果我要翻译成英文、日文啥的，target_lang要传入什么？有没有对照表？

**Comment by zh-plus** [comment, marked as answer]
可以通过https://www.andiamo.co.uk/resources/iso-language-codes/ 查找对应的language-code
